packages/lib-react-components/.npmignore

```diff
@@ -9,3 +9,7 @@ test
 .editorconfig
 .travis.yml
 webpack.dist.js
+
+**/mockLightCurves/*
+**/README.md
```

Comment:
Just to double check - If the Readme files are all excluded, will the Readme instructions at the root level be unintentionally excluded from this package's npm page? https://www.npmjs.com/package/@zooniverse/react-components

Reply:
I think the Readme at the root level is always included with a package. This should just remove the Readme files from individual components in the distributed build.

Reply:
I ran `npm publish --dry-run` for lib-react-components on this branch and the root level Readme is included which is good!

However, the dry run also includes the other Readmes such as `dist/cjs/helpers/withLayer/README.md`. Is that expected?

Reply:
Oh, that's a bug.

Reply:
Oh! 😳 
>The following paths and files are never ignored, so adding them to .npmignore is pointless:
>- package.json
>- README (and its variants)
>- CHANGELOG (and its variants)
>- LICENSE / LICENCE
  
https://docs.npmjs.com/cli/v7/using-npm/developers#keeping-files-out-of-your-package